app/controllers/saml_idp_controller.rb

```diff
@@ -29,7 +29,7 @@ def create
 
   # not using params intentionally
   def idp_make_saml_response(found_user)
-    encode_response found_user, {}
+    encode_response found_user, { signed_message: true }
```

Comment:
why isn't that the default 🤨

Reply:
Broadly, because the SAML spec says it's optional: https://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf#page=68